flink-end-to-end-tests/test-scripts/common_s3_operations.sh

```diff
@@ -150,7 +150,7 @@ function s3_get_number_of_lines_by_prefix() {
 
   # find all files that have the given prefix
   parts=$(aws_cli s3api list-objects --bucket "$IT_CASE_S3_BUCKET" --prefix "$1" |
-    docker run -i --rm stedolan/jq -r '[.Contents[].Key] | join(" ")')
+    docker run -i --rm ghcr.io/jqlang/jq -r '[.Contents[].Key] | join(" ")')
```

Comment:
Would it be worth adding in the version tag instead of using latest? Thinking about test reproducibility.

Reply:
Very good point, thank you!